source/l4_testing/wheel_speeds/wheel_speeds.c

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file wheel_speeds.c
+ * @author Luke Oxley (lcoxley@purdue.edu)
+ * @brief   Use pwm input capture to measure left and right wheel speeds
+ * @version 0.1
+ * @date 2022-01-21
+ * 
+ * @copyright Copyright (c) 2021
+ * 
+ */
+
+#include "wheel_speeds.h"
+#include "can_parse.h"
+
+WheelSpeeds_t wheel_speeds;
+
+uint32_t left_ccr_msb = 0;
+uint32_t left_ccr_lsb = 0;
+uint32_t right_ccr = 0;
```

Comment:
Mask as volatile, could change in IRQ
```suggestion
volatile uint32_t right_ccr = 0;
```